src/main/java/emissary/core/TransformHistory.java

```diff
@@ -193,6 +193,11 @@ public String getKey() {
             return key;
         }
 
+        public String getKeyNoUrl() {
+            String[] splitKey = key.split(".http");
```

Comment:
return StringUtils.substringBefore(key, ".http");

Reply:
Man, I like that helper method. 👍